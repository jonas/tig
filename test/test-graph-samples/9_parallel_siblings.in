commit A Z B C
    Commit A - merge Z, B, and C
commit B D C
    Commit B - merge D and C
commit D E C
    Commit D - merge C and E
commit E Z
    Commit E - after Z
commit Z Y
    Commit Z - after Y
commit Y X
    Commit Y - after X
commit X W
    Commit X - after W
commit W C
    Commit W - after C
commit C
    Commit C
