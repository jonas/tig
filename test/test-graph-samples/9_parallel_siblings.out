●─┬─╮ Commit A - merge Z, B, and C
│ ●─│─╮ Commit B - merge D and C
│ ●─│─│─╮ Commit D - merge C and E
│ ● │ │ │ Commit E - after Z
●─╯ │ │ │ Commit Z - after Y
● ╭─╯ │ │ Commit Y - after X
● │ ╭─╯ │ Commit X - after W
● │ │ ╭─╯ Commit W - after C
◎─┴─┴─╯ Commit C
